.drone.yml

```diff
@@ -87,6 +87,25 @@ steps:
       - sqlite3 /registry/bundles.db "select * from channel" | grep "${DRONE_SEMVER_SHORT}|performance-addon-operator|performance-addon-operator.v${DRONE_SEMVER_SHORT}"
       - sqlite3 /registry/bundles.db "select * from operatorbundle" | grep "\"image\":\"quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator:${DRONE_TAG}\""
 
+  - name: build-must-gather-image
+    image: plugins/docker
+    settings:
+      dockerfile: openshift-ci/Dockerfile.must-gather
+      username:
+        from_secret: QUAY_USER
+      password:
+        from_secret: QUAY_PASSWORD
+      build_args:
+        - COLLECTION_SCRIPTS_DIR=must-gather/collection-scripts
+      registry: quay.io
+      # This only works if github user == quay user!
+      repo: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-must-gather
+
+  - name: validate-registry-image
+    image: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator-must-gather:${DRONE_TAG:-notset}
+    commands:
+      - ls /usr/bin/gather | grep performance
```

Comment:
Is grep for _performance_ correct? I ask since we are copying collection-scripts/* to /usr/bin and _gather_ is a collection script.

Reply:
probably not, I need to figure out a better validation step.